.github/workflows/test.yml

```diff
@@ -75,6 +77,40 @@ jobs:
               DJANGO_DATABASE_ENGINE: 'django.db.backends.mysql'
               DJANGO_DATABASE_PORT: 3306
             continue-on-error: false
+            integration-test-setup-script: |
+              _SETUP_SCRIPT="$(cat << EOM
+                CREATE DATABASE IF NOT EXISTS db;
+                GRANT ALL PRIVILEGES ON db.* TO django;
+                FLUSH PRIVILEGES;
+              EOM
+              )"
+              mysql \
+                --host="${DJANGO_DATABASE_HOST}" \
+                --port="${DJANGO_DATABASE_PORT}" \
+                --user="root" \
+                --password="superpasswd123" \
+                --execute="${_SETUP_SCRIPT}"
+          - python-version: '3.10'
+            django-version: 'Django~=4.1.0'
+            docker-compose-services: maria-db
+            additional-dependencies: mysqlclient
+            env:
+              DJANGO_DATABASE_ENGINE: 'django.db.backends.mysql'
+              DJANGO_DATABASE_PORT: 3307
+            continue-on-error: false
+            integration-test-setup-script: |
```

Comment:
Is there any chance to move this to a file? This way we can have more possible scripts and have less code in the CI file.

Reply:
sure, I will move this script to the file in the next commit